### PR TITLE
Update assign-licenses.md

### PR DESCRIPTION
**Pull request by kmsft**
added link to admin center for ease of navigation.  the URL is not listed anywhere in current docs making it slightly harder for new users

### CODE REVIEW

File: windows-365/enterprise/assign-licenses.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ ms.collection: M365-identity-device-management
 
 Before a user can use a Cloud PC, you must assign a [Windows 365 license](https://www.microsoft.com/windows-365/all-pricing) to that user. You can assign the licenses using either of these methods:
 
-- Microsoft 365 admin center for individual users. For steps on how to use admin center to assign licenses, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).
+- [Microsoft 365 admin center](https://admin.microsoft.com/) for individual users. For steps on how to use admin center to assign licenses, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/assign-licenses.md:35]:
```suggestion
- [Microsoft 365 admin center](https://admin.microsoft.com/) for individual users. For steps on how to use the admin center to assign licenses, see [Assign licenses to users](/microsoft-365/admin/manage/assign-licenses-to-users).
```

File: windows-365/enterprise/assign-licenses.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ ms.collection: M365-identity-device-management
...
 - [Azure AD admin center](https://aad.portal.azure.com/) for group license assignments. For more information about group license assignments, see [Assign licenses to users by group membership in Azure Active Directory](/azure/active-directory/enterprise-users/licensing-groups-assign).
 - To assign direct licenses to a list of individual users, see [Assign licenses for Windows 365](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell) or see [Assign license](/graph/api/user-assignlicense) to perform through Graph
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/assign-licenses.md:37]:
```suggestion
- To assign direct licenses to a list of individual users, see [Assign licenses for Windows 365](/microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell) or see [Assign license](/graph/api/user-assignlicense) to perform it through Graph.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kmsft, kindly commit the additional suggested changes. Many thanks.

File: windows-365/enterprise/assign-licenses.md (hunk 2 above)
Comment (kmsft) [reply windows-365/enterprise/assign-licenses.md:37]: hi @JohanFreelancer9 I would suggest skipping that part of the suggestion as those lines were replaced with a rewrite in https://github.com/MicrosoftDocs/memdocs/pull/3230/files/2bbfd196e31718f3e3beddc533d5ca1a10ad89e2

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
#sign-off

**Comment by ErikjeMS** [comment]
@kmsft Please recreate this PR in the private repo. Closing.

**Pull request closed without merging**